.github/workflows/release.yml

```diff
@@ -31,14 +31,14 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: '^1.13.7'
-      - name: Build Carrier
+      - name: Build Fuseml
```

Comment:
I wonder if we could write it `FuseML` instead of `Fuseml` ?

Reply:
Here yes. In go code probably not.